src/Notifier.js

```diff
@@ -131,6 +133,14 @@ const Notifier = {
             plaf.requestNotificationPermission().done((result) => {
                 if (result !== 'granted') {
                     // The permission request was dismissed or denied
+                    const description = result === 'denied'
+                        ? 'Your browser is not permitting this app to send you notifications.'
+                        : 'It seems you didn\'t accept notifications when your browser asked';
```

Comment:
I suggest:

"Riot does not have permission to send you notifications - please check your browser settings"